block/Kconfig

```diff
@@ -186,6 +186,12 @@ config BLK_CGROUP_IOPRIO
 	scheduler and block devices process requests. Only some I/O schedulers
 	and some block devices support I/O priorities.
 
+config BLK_NODE_LATENCY
+       bool "Track per-node I/O latency"
+       help
```

Comment:
This commit contains the following problems.

```
jmeneghi:linux(newiop6|REBASE-i 4/9) > scripts/checkpatch.pl --git HEAD
WARNING: please write a help paragraph that fully describes the config symbol
#30: FILE: block/Kconfig:189:
+config BLK_NODE_LATENCY
+       bool "Track per-node I/O latency"
+       help
+       Enable per-node I/O latency tracking. This can be used by I/O schedulers
+       to determine the node with the least latency.
+

WARNING: added, moved or deleted file(s), does MAINTAINERS need updating?
#65: 
new file mode 100644

WARNING: line length of 104 exceeds 100 columns
#100: FILE: block/blk-nlatency.c:31:
+       unsigned int lowat;     /* Low Watermark below which latency measurement is deemed unreliable */

WARNING: Missing a blank line after declarations
#401: FILE: block/blk-nlatency.c:332:
+       struct rq_nlat *nlat;
+       if (!rqos)

WARNING: Prefer kcalloc over kzalloc with multiply
#425: FILE: block/blk-nlatency.c:356:
+       nlat->latency = kzalloc(sizeof(u64) * nlat->num, GFP_KERNEL);

WARNING: Prefer kcalloc over kzalloc with multiply
#428: FILE: block/blk-nlatency.c:359:
+       nlat->samples = kzalloc(sizeof(unsigned int) * nlat->num, GFP_KERNEL);

WARNING: ENOTSUPP is not a SUSV4 error code, prefer EOPNOTSUPP
#499: FILE: include/linux/blk-mq.h:1243:
+static inline in blk_nlat_init(struct gendisk *disk) { return -ENOTSUPP; }

total: 0 errors, 7 warnings, 448 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

Commit 79a807940710 ("block: track per-node I/O latency") has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.
```